### PR TITLE
fix help text display issue, translations

### PR DESCRIPTION
**Pull request by fanjinfei**

### CODE REVIEW

**Comment by wardi** [comment]
A better commit message would be "allow HTML to be used in upload_warn messages". Would you make the same change for upload_info as part of this PR too?

**Comment by fanjinfei** [comment]
I might add other changes to help text in the same PR. Should I change the message now?

**Comment by wardi** [comment]
Sure, I'll wait for you to mark this ready for review

**Comment by fanjinfei** [comment]
ready for review

File: ckanext/canada/tables/contracts.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ portal_type: info
 collection: pd
 
 resources:
-- title: Proactive Disclosure - Contracts
+- title: Proactive Disclosure - Contracts over 10K (Quarterly)
```
Comment (wardi) [inline ckanext/canada/tables/contracts.yaml]: need to update the .po as well?

Comment (wardi) [inline ckanext/canada/tables/contracts.yaml]: oh, I see this is the same as the title above, nevermind

File: ckanext/canada/templates/internal/recombinant/snippets/xls_upload.html [hunk 2]
```diff
@@ -15,7 +15,7 @@
   {% set geno = h.recombinant_get_geno(pkg.dataset_type) %}
   {% if 'upload_warn' in geno %}
     <div class="alert alert-danger">
-      <p>{{ _(geno.upload_warn) }}</p>
+      <p>{{ _(geno.upload_warn) | safe }}</p>
```
Comment (wardi) [inline ckanext/canada/templates/internal/recombinant/snippets/xls_upload.html]: please update `upload_info` above the same way, so we can add links there too

**Pull request merged**